CHANGELOG.md

```diff
@@ -27,6 +27,12 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 * hosts: Static hosts now include the host sets they are in.
   ([PR](https://github.com/hashicorp/boundary/pull/1828))
 
+### Bug Fixes
+
+* controller: Fix creating and starting one Controller gRPC Server
```

Comment:
This isn't user facing and doesn't really affect anything non-internal so shouldn't go in the changelog.